### PR TITLE
Fixes #8813 Retain search value after submitting

### LINKED ISSUE #8813: Search bar contents not retained after submitting
### NetBox version

v3.1.8

### Python version

3.8

### Steps to Reproduce

1. Enter a search term into the main search bar and submit
2. See empty search bar

### Expected Behavior

1. Enter a search term into the main search bar and submit
2. Search term should still be present

### Observed Behavior

1. Enter a search term into the main search bar and submit
2. Search bar is empty

### PR DESCRIPTION
**Pull request by PieterL75**

### CODE REVIEW

File: netbox/utilities/templates/search/searchbar.html [hunk 1]
```diff
@@ -5,7 +5,11 @@
     aria-label="Search"
     placeholder="Search"
     class="form-control"
-    value="{{ request.GET.q }}"
+    value=""
```
Comment (jeremystretch) [inline netbox/utilities/templates/search/searchbar.html:8]: Why not leave this as-is?

Comment (PieterL75) [reply netbox/utilities/templates/search/searchbar.html:8]: By setting the value with a javascript, the cursor moves to the end of the input field.. if I just set the value, the cursor is at the start of the input field...

Comment (jeremystretch) [reply netbox/utilities/templates/search/searchbar.html:8]: But it doesn't even render the value until the user clicks the field, which doesn't resolve #8813. Just leaving it as `value="{{ request.GET.q }}"` ensures the value is populated, and the cursor defaults to the end of the string on focus.

Comment (PieterL75) [reply netbox/utilities/templates/search/searchbar.html:8]: The autofocus part takes care of setting the focus. 
If I leave out the onfocus part. then the cursor is at the start of the value.
You need to merge that PR #8908 to make this work. The autofocus is currently overruled by the piece of javascript in the netbox.ts

Comment (jeremystretch) [reply netbox/utilities/templates/search/searchbar.html:8]: Ok. For now, can we change this to `value="{{ request.GET.q }}"` just to close out #8813? I suspect #8908 might need some additional tweaking but I'm waiting for Matt to weigh in. We can add the autofocus bit under that PR to ensure it's correlated with the appropriate FR.

Comment (PieterL75) [reply netbox/utilities/templates/search/searchbar.html]: I removed the autocomplete and onfocus from this PR.. Moving that to the #8908

**Pull request merged**